package_control/package_tasks.py

```diff
@@ -624,7 +631,11 @@ def render_quick_panel_items(self, tasks):
                         final_line += ' '
                     final_line += '<a href="{}">{}</a>'.format(homepage, homepage_display)
 
-                items.append(sublime.QuickPanelItem(task.package_name, [description, final_line]))
+                annotation = ''
+                if task.last_modified:
+                    annotation = datetime.datetime.strptime(task.last_modified, '%Y-%m-%d %H:%M:%S').strftime('Updated at %a %b %d %H:%M:%S %Y')
```

Comment:
I would think that it's excessive with time. Day, month and year would probably be enough?